apps/air-discount-scheme/api/src/environments/environment.ts

```diff
@@ -1,7 +1,7 @@
 export default {
```

Comment:
You put sentry_dsn to web in the helm PR, why aren't you adding `process.env.SENTRY_DSN` to web environment?

Reply:
It's already in next.config.js